docs/can-stache-element.md

```diff
@@ -135,6 +135,33 @@ customElements.define("count-er", Counter);
 @codepen
 @highlight 9-11,only
 
+#### Observable class fields
+
+`StachElement` class fields are also observable.,
```

Comment:
can you add the link to class fields docs on MDN?

Reply:
@phillipskevin Done!